line_profiler/cymem/cymem/about.py

```diff
@@ -0,0 +1,7 @@
+__title__ = "cymem"
+__version__ = "2.0.7"
+__summary__ = "Manage calls to calloc/free through Cython"
+__uri__ = "https://github.com/explosion/cymem"
```

Comment:
was a pip dependency on cymem not suitable here?

Reply:
If I recall correctly, I tried that but Cython couldn't find the pyx code for cymem when compiling. I can try again if you think this is a big issue.